tests/ratio_estimator_test.py

```diff
@@ -81,39 +85,88 @@ def build_classifier(theta, x):
     return RatioEstimator(net=net, theta_shape=theta[0].shape, x_shape=x[0].shape)
 
 
-def build_classifier_missing_args():
+def build_estimator(theta, x):
+    return build_mdn(theta, x)
+
+
+def build_estimator_missing_args():
     pass
 
 
-def build_classifier_missing_return(theta: Tensor, x: Tensor):
+def build_estimator_missing_return(theta: Tensor, x: Tensor):
```

Comment:
seeing this this test now seems to be more general, and not only about NRE. Therefore, we should move this test to, e.g., `density_estimator_test.py`.